tuntap.c

```diff
@@ -58,6 +58,8 @@ tuntap_destroy(struct device *dev) {
 
 char *
 tuntap_get_ifname(struct device *dev) {
+  /* null terminate (just in case) */
+  dev->if_name[IFNAMSIZ] = 0;
```

Comment:
Indentation is wrong, please follow the style of the rest of the file.
null terminate on IFNAMSIZ will truncate the name of the interface if the name is IFNAMSIZ bytes. I think we shall not do that.
We better change the size of if_name to IFNAMSIZ + 1 and ensure null termination on assignment.
What do you think ?